checkstyle.xml

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+	<property name="charset" value="UTF-8"/>
+	<property name="tabWidth" value="4"/>
+    <module name="TreeWalker">
+        <module name="AvoidStarImport">
+            <property name="severity" value="warning" />
+        </module>
+        <module name="ConstantName"/>
+    	<module name="EmptyBlock">
+    		<property name="option" value="TEXT"/>
+      		<property name="tokens" value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE,
+      		                               LITERAL_SWITCH"/>
+    	</module>
+    	<module name="AbstractClassName"/>
+    	<module name="AnnotationLocation"/>
+    	<module name="OneStatementPerLine"/>
+    	<module name="PackageDeclaration"/>
+    	<module name="MissingOverride">
+    		<property name="severity" value="warning" />
+        </module>
+        <module name="LeftCurly">
+  			<property name="ignoreEnums" value="false"/>
+		</module>
+		<module name="NeedBraces">
+  			<property name="tokens" value="LITERAL_IF, LITERAL_ELSE, LITERAL_DO, LITERAL_FOR,
+  			                               LITERAL_WHILE"/>
+		</module>
+		<module name="RightCurly">
+  			<property name="tokens" value="LITERAL_TRY, LITERAL_CATCH, LITERAL_FINALLY, LITERAL_IF,
+  			                               LITERAL_DO"/>
+		</module>
+		<module name="InnerTypeLast"/>
+		<module name="ImportOrder">
+  			<property name="groups" value="/^java\./,javax,org,it.arcade.hospital"/>
+  			<property name="ordered" value="true"/>
+  			<property name="separated" value="true"/>
+  			<property name="option" value="above"/>
+  			<property name="sortStaticImportsAlphabetically" value="true"/>
+		</module>
+		<module name="UnusedImports"/>
+		<module name="InvalidJavadocPosition"/>

```

Comment:
since we don't plan to have Javadoc yet, delete for now all the stuff that is associated with Javadoc

Reply:
I can delete it, but are you sure that any javadocs won't be added in the future?

Reply:
I am sure for the nearest future but not for long term one. The important thing is, that the state of the checkstyle should "correspond" to the current project rules and state.That means, if we for example do not have Javadoc, we shouldn't have javadoc in the checkstyle, because there is no point to have this rules and additional ton of errors that can not be solved. But for now you can as comment it out as well instead of deleting it- so in case they are necessary, they will be there.